package2/scripts/node_b.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+
+import rospy
+from std_msgs.msg import UInt16
+from std_msgs.msg import Float32
+
+rospy.init_node('node_b', anonymous = True)
+
+
+def callback(msg):
+    out = Float32()
+    out = msg.data / 0.15
+    rospy.loginfo(out)
+    pub.publish(out)
+
+sub = rospy.Subscriber('/Beliavskij', UInt16, callback)
+pub = rospy.Publisher('/kthfs/result', Float32, queue_size = 10)
+
+rospy.spin()
```

Comment:
```suggestion
def callback(msg):
    out = Float32()
    out = msg.data / 0.15
    rospy.loginfo(out)
    pub.publish(out)

if __name__ == '__main__':
    rospy.init_node('node_b', anonymous = True)
    
    sub = rospy.Subscriber('/Beliavskij', UInt16, callback)
    pub = rospy.Publisher('/kthfs/result', Float32, queue_size = 10)
    
    rospy.spin()
```

Very small details, but otherwise it works. I would also suggest that you can refactor slightly a bit both these nodes (node_a and node_b) using a class.